public/css/browse.css

```diff
@@ -0,0 +1,12 @@
+.col {
+  margin-top: 2em;
+}
+
+.row a {
```

Comment:
@abhinav098 Please refrain from using bootstrap classes for adding CSS
<img width="878" alt="Screen Shot 2020-11-23 at 10 49 52 AM" src="https://user-images.githubusercontent.com/20966155/99983265-a48e1600-2d79-11eb-883c-5ba1b8e7a46a.png">
 properties. Add your own classes/ids. 
If we use bootstrap classes, it will override the original CSS applied by bootstrap

Reply:
sure @SonaliChavan95 . Thanks :)